doc/reference/bluetooth/mesh/shell.rst

```diff
@@ -128,10 +128,11 @@ General configuration
 	Initialize the mesh. This command must be run before any other mesh command.
 
 
-``mesh reset``
---------------
+``mesh reset <value: local, remote>``
+-------------------------------------
 
-	Reset the mesh node to its initial unprovisioned state.
+	Reset the local mesh node to its initial unprovisioned state or reset a remote node and remove it from the network.
+	* ``value``: Sets local node or remote node.
```

Comment:
The local/remote decision is made by the `dst` address in all other APIs, I think this should follow the same pattern. 
The command should also be moved into the Configuration client section.